worker/games.py

```diff
@@ -70,6 +73,28 @@ def is_64bit():
 MAKE_CMD = "make COMP=mingw " if IS_WINDOWS else "make COMP=gcc "
 
 
+def log(s):
+    logfile = os.path.join(os.path.dirname(os.path.realpath(__file__)), LOGFILE)
+    with LOG_LOCK:
+        with open(logfile, "a") as f:
+            f.write("{} : {}\n".format(datetime.datetime.utcnow(), s))
+
+
+def backup_log():
+    try:
+        logfile = os.path.join(os.path.dirname(os.path.realpath(__file__)), LOGFILE)
```

Comment:
`with LOG_LOCK:` ?

Reply:
Ok (the heartbeat thread isn't running yet when this is invoked, but it is better to be safe than sorry).

Reply:
Fixed.